ha_mroonga.cpp

```diff
@@ -14953,7 +14953,105 @@ mrn_io_and_cpu_cost ha_mroonga::scan_time()
   DBUG_RETURN(time);
 }
 
-#ifdef MRN_ENABLE_WRAPPER_MODE
+#ifdef MRN_HANDLER_HAVE_KEYREAD_TIME
+#  ifdef MRN_ENABLE_WRAPPER_MODE
+IO_AND_CPU_COST ha_mroonga::wrapper_rnd_pos_time(ha_rows rows)
+{
+  MRN_DBUG_ENTER_METHOD();
+  IO_AND_CPU_COST res;
+  MRN_SET_WRAP_SHARE_KEY(share, table->s);
+  MRN_SET_WRAP_TABLE_KEY(this, table);
+  res = wrap_handler->rnd_pos_time(rows);
+  MRN_SET_BASE_SHARE_KEY(share, table->s);
+  MRN_SET_BASE_TABLE_KEY(this, table);
+  DBUG_RETURN(res);
+}
+#  endif
+
+IO_AND_CPU_COST ha_mroonga::rnd_pos_time(ha_rows rows)
+{
+  MRN_DBUG_ENTER_METHOD();
+  IO_AND_CPU_COST time;
+#  ifdef MRN_ENABLE_WRAPPER_MODE
+  if (share->wrapper_mode)
+  {
+    time = wrapper_rnd_pos_time(rows);
+  } else {
+#  endif
+    time = handler::rnd_pos_time(rows);
```

Comment:
Could you define `storage_rnd_pos_time()` like `storage_keyread_time()`?

Reply:
I defined `storage_rnd_pos_time()`.